README.md

```diff
@@ -19,16 +18,92 @@
+3. Create and connect another _IB outlet_ to the second segmented control. This should also be of type `UISegmentedControl`.
+4. Connect _both_ segmented controls to the `colorSelected(_:)` _IB action_ already declared in `ViewController.swift`. You should notice something interesting here: More than one UI element can have the _same_ action. That is, you can connect _two_ segmented controls to `colorSelected(_:)`. You haven't seen this happen in previous labs, but this is totally fine, and, in fact, frequently happens in iOS interfaces.
+5. `colorSelected(_:)` is called whenever the user presses one of the buttons in either segmented control. You should write some logic to get the color selected by _each_ control, "mix" them together to produce a new color, and then update the paint bucket to display the mixed color.
+6. Your view controller has a method stubbed out named `mixColors(withFirst:second:)`. Write the logic in this method to mix both the `first` and `second` colors together. This method should return a `String` with the name of the new color ("red", "purple", "green", etc.). You'll have to drawn on your knowledge of if/else statements and switch statements from previous units to complete this method.
+
+### Notes & Hints
 
-*  I think this lab is more about the student solving some simple algorithm where the UI takes more of a backseat. Possibly just dragging over a few labels, buttons to display what it is the function should be spitting back based on certain input. Maybe?? 
-* Could also be like the [Simon Says Lab](https://github.com/learn-co-curriculum/swift-outlets-lab) 
-* I'm ok with it being simple as well just having them re-create 3-5 ViewControllers. Meaning, we show them 3-5 screenshots where each screenshot reprensts a complete ViewController. We just show them the screenshot and a list of the UI elements that make up that view and it's just on them to drag and drop things and change colors to make it look like the particular view. Too easy?
+A few pieces of code have been provided for you to make this lab a bit easier.
 
+1. You can call `color` on a segmented control to get its selected color.
+2. You can call `name` on a color to get the name of the color, as a string.
+3. You can combine \#1 and \#2 to get the name of a color represented by a segmented control. For example, given a segmented control named `sender`, you can call `sender.color.name` to get the name of its selected color.
+4. You can set the color of a view, such as the paint bucket view, with a string using `paintColorName`. For example, if your paint bucket's IB outlet is named `paintBucket`, you can use `paintBucket.paintColorName = "red"` to set the paint color to red.
+5. You can also get the current paint color name by calling `paintBucket.paintColorName` (if `paintBucket` is your IB outlet to the paint bucket).
 
+You should be able to combine all these hints, along with the existing code, to complete this lab. Good luck!
```

Comment:
One thing we can add here (similar to a note we added to a previous lab) is stating something like..

Run and RUN often! Don't write a lot of code assuming everything is working then moving onto another task. Tackle each problem in small chunks adding print functions to make sure your assumptions are correct before moving on.

^^ something like that, but said more poetically I guess.